pytorch_lightning/trainer/connectors/accelerator_connector.py

```diff
@@ -291,6 +294,13 @@ def _check_config_and_set_final_flags(
                     self.checkpoint_io = plugin
                 elif isinstance(plugin, ClusterEnvironment):
                     self._cluster_environment_flag = plugin
+                elif isinstance(plugin, LayerSync):
+                    if sync_batchnorm and not isinstance(plugin, NativeSyncBatchNorm):
```

Comment:
Shouldn't this check the base interface?

```suggestion
                    if sync_batchnorm and not isinstance(plugin, LayerSync):
```

Reply:
no because I want to express here that `Trainer(sync_batchnorm=True, plugins= NativeSyncBatchNorm()) `is valid and should not error, while any other conflicting choice would create an error.